karma.conf.maker.js

```diff
@@ -32,8 +32,8 @@ function newPluginsArray(browserstack) {
     'karma-chrome-launcher',
     'karma-coverage-istanbul-reporter',
     'karma-es5-shim',
-    'karma-expect',
     'karma-mocha',
+    'karma-chai',
```

Comment:
are these changes required by the PR or just included for cleanup?

Reply:
Both.  I removed `karma-expect` and replaced it with `karma-chai` as that is what most tests were already using (I think just had to update some of our video tests to make it work), so that part was cleanup.  But I also used the more advanced object diffing this made available in the new tests for rubicon analytics, so now it's required.